apps/banglerun/src/gps.ts

```diff
@@ -80,8 +91,8 @@ function updateGps(state: AppState): void {
   const pError = dpMag + state.dop * POS_ACCURACY;
   const vError = dvMag + state.dop * VEL_ACCURACY;
 
-  const pGain = state.pError / (state.pError + pError);
-  const vGain = state.vError / (state.vError + vError);
+  const pGain = (state.pError / (state.pError + pError)) || 0;
```

Comment:
These should avoid the occurrence of NaN values in the remote case of a division by 0